tests/cpp_api/CMakeLists.txt

```diff
@@ -60,27 +100,27 @@ add_test(NAME basic_test_rgb
 #                              "${CMAKE_CURRENT_BINARY_DIR}/dataloader"
 #            --build-generator "${CMAKE_GENERATOR}"
 #            --test-command "dataloader"
-#            ${CMAKE_SOURCE_DIR}/data/images/AMD-tinyDataSet
+#            ${ROCM_PATH}/share/rocal/test/data/images/AMD-tinyDataSet
 #)
 
 # dataloader_multithread
 add_test(
   NAME
-  rocAL_dataloader_mt_cpu
+  dataloader_mt_cpu
```

Comment:
@kiritigowda can you name this dataloader_multithread_cpu like in line 118 for gpu